### ISSUE TITLE
Option for just the digits with decimals if possible

### ISSUE THREAD

**Issue by EVIGMA** [cosmicquahog/XpGrapher#6]
Great plugin. Any possibility to have an option to not have graph, just the real-time xp in digit form (like the pre-existing on-canvas tracker by RL). But instead of average, it tracks real-time with 1-2 decimal points if possible.

It would make skills like slayer, where xp varies drastically task to task easier to track. At the moment if I want my xp for a task I have to reset the RL tracker which is annoying, especially if I want to see my total xp gained.

TL;DR: an on canvas xp tracker (like the default one) which instead tracks real-time xp within 2 decimal places (if possible) <3
